src/main/java/de/srendi/advancedperipherals/common/addons/ae2/CraftJob.java

```diff
@@ -140,12 +142,9 @@ public void startCalculation() {
         if (startedCalculation) {
             return;
         }
+        startedCalculation = true;
 
         IGrid grid = node.getGrid();
-        if (grid == null) { //true when the block is not connected
```

Comment:
AE2 source code prevent `getGrid` to return a `null`, I didn't check the version above 1.19.2 tho.
```java
    @Override
    public IGrid getGrid() {
        if (this.myGrid == null) {
            throw new IllegalStateException("A node is being used after it has been destroyed.");
        }
        return this.myGrid;
    }
```

Reply:
👍🏻